internal/cmd/operator-sdk/generate/internal/relatedimages.go

```diff
@@ -26,14 +26,16 @@ import (
 )
 
 // FindRelatedImages looks in the controller manager's environment for images used by the operator.
-func FindRelatedImages(manifestCol *collector.Manifests) ([]operatorsv1alpha1.RelatedImage, error) {
+func FindRelatedImages(manifestCol *collector.Manifests, packageName string) ([]operatorsv1alpha1.RelatedImage, error) {
```

Comment:
I don't like passing in the `packageName` here then using it to detect the base manifest. The function you added below could potentiall produce a false positive since it does not enforce the base CSV being found in the `bases` directory [as sdk does in the logic that calls this function][https://github.com/operator-framework/operator-sdk/blob/master/internal/cmd/operator-sdk/generate/bundle/bundle.go#L183]. We should either make the manifest collector aware of the base CSV so that way you can call `manifestCol.GetBaseClusterServiceVersion()` or pass in `overrides map[string]string` such that its a map of all relatedImages found in the root of the generate command.